docs/aws/audit/shieldmonitoring/rules/guardduty_non_archived_findings_remediation.mdx

```diff
@@ -1,6 +1,220 @@
+</Accordion>
+
+</AccordionGroup>
+</Tab>
+<Tab title='Cause'>
+### Check Cause
+<AccordionGroup>
+<Accordion title='Using Console' defaultOpen='true'>
+1. Sign in to the AWS Management Console and open the Amazon GuardDuty console at https://console.aws.amazon.com/guardduty/.
+
+2. In the navigation pane, choose "Settings".
+
+3. Under "Finding Publishing Frequency", check if "Auto-archive findings" is enabled or not. If it is enabled, it means that the findings are being archived automatically. If it is not enabled, it means that the findings are not being archived.
+
+4. You can also check the status of the "Non Archived Findings" by going to the "Findings" section in the navigation pane. If there are any findings that are not archived, they will be listed here.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. **Install and Configure AWS CLI**: Before you can start using AWS CLI, you need to install it on your local system. You can download it from the official AWS website. After installation, you need to configure it with your AWS account credentials. You can do this by running the command `aws configure` and then entering your AWS Access Key ID, Secret Access Key, Default region name, and Default output format when prompted.
+
+2. **List GuardDuty Detectors**: The first step to check if Non Archived Findings are enabled for GuardDuty is to list all the GuardDuty detectors in your AWS account. You can do this by running the following command: `aws guardduty list-detectors`. This command will return a list of detector IDs.
+
+3. **Get Detector Details**: For each detector ID returned in the previous step, you can get the details of the detector by running the following command: `aws guardduty get-detector --detector-id <detector-id>`. Replace `<detector-id>` with the actual detector ID. This command will return the details of the detector including the status of the "Finding Publishing Frequency" which indicates if Non Archived Findings are enabled or not.
+
+4. **Check Finding Publishing Frequency**: In the output of the previous command, look for the "FindingPublishingFrequency" field. If the value of this field is "FIFTEEN_MINUTES", "ONE_HOUR", or "SIX_HOURS", it means that Non Archived Findings are enabled. If the value is "TWENTY_FOUR_HOURS" or "FORTY_EIGHT_HOURS", it means that Non Archived Findings are not enabled.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. **Import necessary libraries and establish a session**: To start with, you need to import the necessary libraries in your Python script. Boto3 is the Amazon Web Services (AWS) Software Development Kit (SDK) for Python, which allows Python developers to write software that makes use of AWS services like Amazon S3, Amazon EC2, etc. Here is how you can do it:
+
+```python
+import boto3
+import json
+
+# Create a session using your AWS credentials
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>